stable/kured/values.yaml

```diff
@@ -16,8 +16,17 @@ serviceAccount:
   create: true
   name:
 
+autolock:
+  enabled: false
+  image:
+    repository: honestica/k8s-tools
```

Comment:
if there is a better image to used, we can switch.

Reply:
Velero uses `docker.io/bitnami/kubectl:1.14.1`, which is maybe better? (see https://github.com/helm/charts/blob/master/stable/velero/templates/cleanup-crds.yaml)